scss/_list-group.scss

```diff
@@ -173,6 +173,26 @@
   }
 }
 
+// Horizontal flush list items
+//
+// Remove borders and border-radius to keep horizontal list group items edge-to-edge. Most
+// useful within other components (e.g., cards).
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-up($breakpoint) {
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
+
+    .list-group-flush.list-group-horizontal#{$infix} {
+
+      > .list-group-item {
+        border-width: 0 var(--#{$prefix}list-group-border-width) 0 0;
+
+        &:last-child {
+          border-right-width: 0;
+        }
+      }
+    }
+  }
+}
```

Comment:
Could this all be handled in the same grid map-keys function as `.list-group-horizontal`?

Reply:
> Could this all be handled in the same grid map-keys function as `.list-group-horizontal`?

Sure.
I didn't want to mix them on purpose, but if you think it would be preferable, no problem. I've updated the PR. Take a look, please.